.github/workflows/functional-test-noncloud.yaml

```diff
@@ -216,6 +219,11 @@ jobs:
           registry-server: ${{ env.LOCAL_REGISTRY_SERVER }}
           registry-port: ${{ env.LOCAL_REGISTRY_PORT }}
 
+      - name: Install dapr into cluster
```

Comment:
I think we can check if Dapr has both cloud and non-cloud tests and then decide in which workflow (or both) to include this code.

Reply:
I think dapr is only used for noncloud tests, so I removed it in the cloud tests